packages/lib/src/components/Twint/Twint.tsx

```diff
@@ -0,0 +1,47 @@
+import RedirectElement from "../Redirect";
+import PayButton, {payAmountLabel} from "../internal/PayButton";
+import {h} from "preact";
+
+/**
+ * TwintElement
+ */
+
+class TwintElement extends RedirectElement {
+    public static type = 'twint';
+
+    public static defaultProps = {
+        type: TwintElement.type,
+        showPayButton: true
+    };
+    /**
+     * Get the element displayable name
+     */
+    get displayName(): string {
+        const { i18n, name, storedPaymentMethodId } = this.props;
+        return storedPaymentMethodId ? `${name} ${i18n.get('twint.saved')}`
```

Comment:
Does 'twint.saved' exist in the translations file already?

Reply:
No, not yet

Reply:
Updated the files